lib/decompress/zstd_decompress.c

```diff
@@ -114,7 +114,6 @@ typedef struct {
     ZSTD_seqSymbol OFTable[SEQSYMBOL_TABLE_SIZE(OffFSELog)];
```

Comment:
Can we add a comment here saying that all data up to the `hufTable` may be overwritten when constructing the huffman table.

Reply:
That's only true when building `DDict`,
because the content of these tables is not yet initialized, 
and is ensured, by construction, to be initialized after huffman.

However, this assumption is no longer correct in a "normal" `DCtx` context,
as FSE tables might contain valid distributions, to be re-used in next blocks.